tests/publiccloud/az_accelerated_net.pm

```diff
@@ -64,7 +62,7 @@ sub enable_accelerated_net {
     assert_script_run("az network nic update --name $name-nic --resource-group $name --accelerated-networking true", timeout => 60 * 10);
     assert_script_run("az vm start --resource-group $name --name $name",                                             timeout => 60 * 20);
     sleep 60 * 3;    # Sometimes, IP is not reachable after the restart and 5 minutes is enough.
-    $instance->{ip} = $self->get_new_ip($instance);
+    $instance->ip($self->get_new_ip($instance));
```

Comment:
not obligatory in terms of this PR , but maybe `get_new_ip` also should be method  of `publiccloud::instance` ?